packages/@aws-cdk/aws-codepipeline-actions/lib/ecs/deploy-action.ts

```diff
@@ -58,7 +67,13 @@ export class EcsDeployAction extends Action {
       resource: props.service,
     });
 
+    const deploymentTimeout: number | undefined = props.deploymentTimeout && props.deploymentTimeout.toMinutes({ integral: true });
+    if (typeof deploymentTimeout !== 'undefined' && (deploymentTimeout < 1 || deploymentTimeout > 60)) {
```

Comment:
I would just have `deploymentTimeout !== undefined` here instead of the `typeof` check.

Reply:
Done!